### ISSUE TITLE
Replace the tab / popup with a panel attached to the button.

### ISSUE THREAD

**Issue by Aldarone** [wallabag/firefox-ext#38]
Considering all the issues with tab autoclosing working but not the popup and/or vice versa I'm thinking that a panel would be more suitable: https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/panel#Attaching_panels_to_buttons

![panel-button](https://cloud.githubusercontent.com/assets/2682011/12848444/3d14626a-cc1a-11e5-8449-6cad53ffddf6.png)

I didn't use one for the Shaarli extension because panels close themselves when the user clicks outside and it was a bit counterintuitive for this use case but I don't think it matters for wallabag.

What do you think ?

**Comment by stepardo** [comment]
On 05.02.2016 15:08, Alda Marteau-Hardi wrote:
> Considering all the issues with tab autoclosing working but not the
> popup and/or vice versa I'm thinking that a panel would be more
> suitable:
> https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/panel#Attaching_panels_to_buttons
> 
> panel-button
> <https://cloud.githubusercontent.com/assets/2682011/12848444/3d14626a-cc1a-11e5-8449-6cad53ffddf6.png>
> 
> I didn't use one for the Shaarli extension because panels close
> themselves when the user clicks outside and it was a bit
> counterintuitive for this use case but I don't think it matters for
> wallabag.
> 
> What do you think ?

Do you want it to look and function like the social service?

**Comment by Aldarone** [comment]
Yes it'll be very close. But the social service has two downsizes : 

- We must wait for Wallabag v2
- It only works on HTTPS

**Comment by nicosomb** [comment]
About wallabag v2, is this addon ready?

**Comment by tcitworld** [comment]
Not at all, since we don't have chosen a auth way for the API. :/

**Comment by nicosomb** [comment]
But we can simulate the same behavior than wallabag v1: redirect to wallabag with the URL in parameter. 
And then, we can enhance this addon with authentication.?

**Comment by tcitworld** [comment]
Yeah, well, we can make a checkbox for wallabag v2 (the rss feed url is different), sure.

**Comment by nicosomb** [comment]
:+1: Go! Who can do that?
